lib/config/mssql.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/artie-labs/transfer/lib/config/constants"
+	"github.com/artie-labs/transfer/lib/stringutil"
+)
+
+type MSSQL struct {
+	Host     string `yaml:"host"`
+	Port     int    `yaml:"port"`
+	Username string `yaml:"username"`
+	Password string `yaml:"password"`
+	Database string `yaml:"database"`
+}
+
+func (m *MSSQL) DSN() string {
+	return fmt.Sprintf("sqlserver://%s:%s@%s:%d?database=%s", m.Username, m.Password, m.Host, m.Port, m.Database)
```

Comment:
Does database need to be query string encoded?

Reply:
I believe it does need to be encoded, I changed the DSN format.